style.css

```diff
@@ -78,3 +78,381 @@ span {
+  height: 45px;
+  width: 159px;
+  background: #36b37f;
+  border-radius: 4px;
+  border-style: none;
+}
+
+.about {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.description {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  letter-spacing: normal;
+  font-size: 15px;
+  line-height: 28px;
+  text-align: center;
+  color: #42526e;
+}
+
+.coding-languages {
+  display: flex;
+  width: 290px;
+  height: 352px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.coding-languages > div {
+  margin: auto;
+}
+
+.btn-group1 {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.languages1 {
+  color: #36b37e;
+  height: 40px;
+  width: 60px;
+  border-style: none;
+  border-radius: 6px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 10px;
+  background-color: white;
+  margin: 5px;
+}
+
+.languages1 + .languages1 {
+  margin-left: 25%;
+  margin-left: 3px;
+}
+
+.frameworks {
+  display: flex;
+  flex: auto;
+  height: 352px;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.frameworks > div {
+  margin: auto;
+}
+
+.btn-group2 {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+}
+
+.languages2 {
+  color: #36b37e;
+  height: 40px;
+  width: auto;
+  border-style: none;
+  border-radius: 6px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 10px;
+  background-color: white;
+  margin: 5px;
+}
+
+.languages2 + .languages2 {
+  margin-left: 25%;
+  margin-left: 3px;
+}
+
+.skills {
+  display: flex;
+  width: 290px;
+  height: 352px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.skills > div {
+  margin: auto;
+}
+
+.btn-group3 {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.languages3 {
+  color: #36b37e;
+  height: 40px;
+  width: 60px;
+  border-style: none;
+  border-radius: 6px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 10px;
+  background-color: white;
+  margin: 5px;
+}
+
+.languages3 + .languages3 {
+  margin-left: 25%;
+  margin-left: 3px;
+}
+
+.lang-icon {
+  width: 80px;
+  height: 80px;
+  background: #ebf0ee;
+  box-sizing: border-box;
+}
+
+.res {
+  font-family: "Inter", sans-serif;
+  margin-top: 50px;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 32px;
+  text-align: center;
+  color: #344563;
+}
+
+.wrapper {
+  display: grid;
+  gap: 10px;
+}
```

Comment:
- [ ] Nice you are displaying our main container as grid as suggested however there are missing some code in this called class to meet the design, refer to this [link](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiQw4SZp67yAhXOs6QKHTxXBRgQFnoECAYQAw&url=https%3A%2F%2Fcoryrylan.com%2Fblog%2Fhow-to-center-in-css-with-css-grid&usg=AOvVaw08xt_w55kt7GmqbtLFpxNN) for more details